code/game/machinery/OpTable.dm

```diff
@@ -100,10 +100,14 @@
 	if(!anes_tank)
 		to_chat(user, SPAN_WARNING("There is no anesthetic tank connected to the table, load one first."))
 		return
-	if(H.wear_mask && !H.drop_inv_item_on_ground(H.wear_mask))
-		to_chat(user, SPAN_DANGER("You can't remove their mask!"))
-		return
-
+	if(H.wear_mask)
+		var/obj/item/W = H.wear_mask
+		if(W.flags_inventory & CANTSTRIP)
+			var/obj/item/clothing/mask/facehugger/hugger = W
```

Comment:
There should be an explicit check for if the mask is a facehugger before assigning it. While CANTSTRIP has a comment saying it's only used for facehuggers this may change in the future and there are items with CANTSTRIP already.

Reply:
if those items cannot be stripped by normal means they should not be able to be stripped by the surgery table anyways so. think of it as a double bugfix

Reply:
the reason why i used the CANSTRIP instead of (ISTYPE) is because in the future some mask with that flag may be added and instead of having to make an exception for every single one we better just do this

Reply:
This needs to be rethought if it's going to be generalized.